app/models/game.rb

```diff
@@ -51,4 +51,22 @@ def populate_board! # rubocop:disable Metrics/AbcSize
 
     pieces.create(color: 'black', type: 'King', current_row_index: 7, current_column_index: 4)
   end
+
+  def forfeit(current_user_id)
+    current_user = User.find(current_user_id)
+    if current_user.id == white_player_id
+      if black_player_id.nil?
```

Comment:
This line could be if
 if !black_player_id.nil?
 then the logic, that way there is no else statement.